### PR TITLE
[HIRFUS-110] Product Array Algorithm

### PR DESCRIPTION
**Pull request by jomlink**
commit on array product price algorithms

### CODE REVIEW

File: submissions/jonasMwansa/data structures and algorithms 1/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="index.js" type="text/javascript"></script>
```
Comment (ToluEnny24) [inline submissions/jonasMwansa/data structures and algorithms 1/index.html:7]: It is bad coding practice to have your script in the <head> tag.

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
@jomlink please effect the requested changes

File: submissions/jonasMwansa/data structures and algorithms 1/index.html (hunk 1 above)
Comment (jomlink) [reply submissions/jonasMwansa/data structures and algorithms 1/index.html:7]: I have just noticed I placed it in the head tag,
let me place it at the bottom of the body, thank you @ToluEnny24

**Comment by ToluEnny24** [comment]
@jomlink Changes seen. I will close this so you can recreate and use a correct PR Title.

**Pull request closed without merging**